src/navigation/menu.scss

```diff
@@ -33,11 +33,15 @@
     &::before { border-bottom-left-radius: $border-radius; }
   }
 
-  &:focus,
+  &:focus {
+    z-index: 1;
+    outline: none;
+    box-shadow: var(--color-state-focus-shadow);
+  }
```

Comment:
This might be a bigger accessibility question but should/could we make this a default for all focusable elements?

Reply:
Hmm.. yeah. Currently we have a mix between native focus (e.g. links) and custom focus (input, button). It would be nice to unify it:

```css
*:focus {
  outline: none;
  box-shadow: var(--color-state-focus-shadow);
}
```

![focus-2](https://user-images.githubusercontent.com/378023/118112838-80ee8300-b420-11eb-9e7b-9cd7513d141f.gif)

But we'd have to tweak existing components like the underline nav:

![Screen Shot 2021-05-13 at 19 28 48](https://user-images.githubusercontent.com/378023/118113761-a7f98480-b421-11eb-8a65-a8ec5a3e473c.png)

Another option might be to only change the outline color: 

```css
*:focus {
  outline-color: var(--color-border-info);
}
```

![focus-3](https://user-images.githubusercontent.com/378023/118112948-a3809c00-b420-11eb-93b1-6d1ee490a0db.gif)

Reply:
Added an issue here: https://github.com/github/primer/issues/160